Sources/GoogleMapsUtils/Geometry/Models/GMUPlacemark1.swift

```diff
@@ -14,18 +14,18 @@
 
 /// Represents a placemark which is either a Point, LineString, Polygon, or MultiGeometry. Contains
 /// the properties and styles of the place.
-/// TO-DO: Rename the class to `GMUPlacemarkSwift & GMUGeometryContainer` once the linking is done and remove the objective c class.
+/// TO-DO: Rename the class to `GMUPlacemarkSwift` once the linking is done and remove the objective c class.
```

Comment:
We should just rename it to `GMUPlacemark` (without the Swift suffix)

Reply:
Oops! I'll rename in next PR!